parsec/core/gui/devices_widget.py

```diff
@@ -51,7 +51,7 @@ def show_context_menu(self, pos):
         menu.exec_(global_pos)
 
     def change_password(self):
-        self.change_password_clicked.emit(self.device_name)
+        self.change_password_clicked.emit(self.device_info.device_label)
```

Comment:
The argument is not actually used (since it's only possible to change the password of the current device) and can be removed.

Reply:
good point !

Reply:
fixed